test/tag.http

```diff
@@ -0,0 +1,25 @@
+GET http://localhost3000/tags
+
+###
+
+POST http://localhost3000/tags
+Content-Type: application/json
+{
+  instrument: "flute de Pan"
+  status: "Amateur",
+  style: "Alternatif"
+}
+
+###
```

Comment:
this deserves a true comment (not just "###") to know what this request does :)

Reply:
Fixed, now has a comment.